paseos/actors/ground_station_actor.py

```diff
@@ -16,3 +27,21 @@ def __init__(self, name: str, epoch: pk.epoch) -> None:
         """
         logger.trace("Instantiating GroundstationActor.")
         super().__init__(name, epoch)
+
+    def get_position(self, epoch: pk.epoch):
+        """Compute the position of this ground station at a specific time.
```

Comment:
Position with respect to what? Is this supposed to change over time because the central body could be not Earth?

Reply:
Everything is in J2000, same reference frame as for the spacecraft actors. Since the Earth is rotating (but the frame isn't), ground stations have a non-constant position depending on time. Added more comment to explain.